xapian-core/include/xapian/mset.h

```diff
@@ -74,6 +80,38 @@ class XAPIAN_VISIBILITY_DEFAULT MSet {
     /// Destructor.
     ~MSet();
 
+    /** Assigns new weights and updates MSet.
```

Comment:
:+1: have added a line but since I have not changed line 83 it is not showing outdated